app/src/main/java/io/github/droidkaigi/confsched2018/presentation/feed/item/FeedItem.kt

```diff
@@ -4,58 +4,68 @@ import android.support.constraint.ConstraintSet
 import android.support.transition.Transition
 import android.support.transition.TransitionListenerAdapter
 import android.support.transition.TransitionManager
+import android.support.v4.content.res.ResourcesCompat
 import android.support.v7.widget.RecyclerView
 import android.view.View
 import android.view.ViewTreeObserver
 import com.xwray.groupie.databinding.BindableItem
 import io.github.droidkaigi.confsched2018.R
 import io.github.droidkaigi.confsched2018.databinding.ItemFeedBinding
 import io.github.droidkaigi.confsched2018.model.Post
+import io.github.droidkaigi.confsched2018.util.ext.context
 
 data class FeedItem(
         val post: Post,
         private val feedItemCollapsed: ConstraintSet,
         private val feedItemExpanded: ConstraintSet,
         private val expandTransition: Transition,
-        private val collapseTransition: Transition,
-        private var expanded: Boolean = false
+        private val collapseTransition: Transition
 ) : BindableItem<ItemFeedBinding>(
         post.hashCode().toLong()
 ) {
-    private val touchIgnorer = View.OnTouchListener { _, _ -> true }
+    private val touchIgnore = View.OnTouchListener { _, _ -> true }
+    private var initialized: Boolean = false
+    private var expanded: Boolean = false
 
     override fun bind(viewBinding: ItemFeedBinding, position: Int) {
         viewBinding.post = post
 
+        viewBinding.feedIcon.setImageResource(when (post.type) {
```

Comment:
Can you change the type to enum?
https://github.com/droidkaigi/conference-app-2018/blob/9c045ad14eef92ade7507a660f24156b12d7a110/app/src/main/java/io/github/droidkaigi/confsched2018/data/api/response/mapper/NoticeDataMapperExt.kt#L13-L13

Reply:
@takahirom How about? 01e18317597b05b5e2c148539eaf52cfecb1cdad

Reply:
GREAT 💯